wiki/en/Running-a-Server.md

```diff
@@ -298,6 +297,49 @@ You yourself should connect using the local network (LAN) address of the machine
 
 Most domestic internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your specic set up. Your router might support some "dynamic DNS" providers out of the box. If this is not the case, set up a dynamic DNS client as described by the dynamic DNS provider you chose.
 
+## DNS SRV record support
+
+Jamulus client supports [DNS SRV records ("service" records)](https://en.wikipedia.org/wiki/SRV_record). If Jamulus client finds an SRV record associated with the domain name given in the Connect window, it will try to connect to the endpoint host and port listed in the SRV record. This process is similar to HTTP redirection in a web browser, only it's between Jamulus client and the DNS server. Jamulus server is not involved in this interaction. Jamulus server just listens for connections on the same port listed in the SRV record.
+
+#### Why is this helpful?
+
+Jamulus client can connect to a server on a non-standard port by specifying the port as part of the server address. A server administrator may want to give users a simple address without the port details, while still serving Jamulus on a non-standard port. For this case, the port information can be acquired from an SRV record.
```

Comment:
```suggestion
A Jamulus client can connect to a Server on a non-standard port by specifying the port as part of the server address. A server administrator may want to give users a simple address without the port details, while still serving Jamulus on a non-standard port. For this case, the port information can be acquired from an SRV record.
```

@pljones and @gilgongo had a discussion about Server vs server somewhere, I think. We should stay consistent.

Reply:
I think so, but I'm not sure.

Reply:
> I try to maintain consistency per-page, because differing conventions co-exist in this publication. Does this whole page capitalize Server, or naw?

I see the style guide is out of date on this, but as of a couple of releases ago, we capitalise "Server" when we mean the Jamulus software running in that mode This is to distinguish it from the machine on which it runs. Eg "I am running two Servers on my AWS server"